enclave_manager/avalon_enclave_manager/kme/kme_enclave_info.py

```diff
@@ -183,7 +185,12 @@ def _init_enclave_with(self, signed_enclave, config):
         Returns :
             @returns tcf_enclave_info - An instance of the tcf_enclave_info
         """
+        # Get sealed data if persisted from previous startup.
+        persisted_sealed_data = file_utils.read_file(
```

Comment:
Shouldn't this be conditional. How do we start the KME if we don't want to initialize from sealed storage ?

Reply:
This should be the default behavior and kind of mandatory as well. Exposing a config to have a choice will make the system vulnerable. We can put a FAQ to delete the sealed data in case such a need arises. I do not foresee a straightforward case where this would be required.

Reply:
Please update the documentation to explain how to start KME/Singleton enclave with new worker signing and encryption key.